### PR TITLE
Added tests for Team model methods

### PR DESCRIPTION
**Pull request by makarman**
Added test for to_dict method
Added test for create method
Added test for update method
Added test for get_by_id method
Added test for delete_by_id method

## Code reviewers

- [ ] @dmytrorevak
- [ ] @IvanAndrukh
- [ ] @JjuliaP
- [ ] @makarman
- [ ] @noobhunters
- [ ] @Silent777
- [ ] @Staryj812

### Second Level Review

- [ ] @ssOleg
- [ ] @lhalam

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: eventually/tests/unittests/team/testmodel.py [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+            self.team3 = Team(103, owner=owner, members=members, name=name, description=description, image=image)
+            self.team3.save()
+
+    def test_team_to_dict(self):
+        """Method that tests `to_dict` method of certain Team instance."""
+
+        test_team_dict1 = {
+            'id': 101,
+            'name': 'bananaboys',
+            'description': 'krakaziabra',
```
Comment (ssOleg) [inline eventually/tests/unittests/team/testmodel.py]: Please change all strange strings like `bananaboys` and `krakaziabra` to `some name`, `some description` ...

File: eventually/tests/unittests/team/testmodel.py [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+            'created_at': 1508044512,
+            'updated_at': 1508044512,
+            'owner_id': 101,
+            'members_id': [101, 102, 103]
+        }
+
+        test_team_dict2 = {
+            'id': 102,
+            'name': 'bananadogs',
+            'description': 'krakaziabra mama',
```
Comment (ssOleg) [inline eventually/tests/unittests/team/testmodel.py]: for example, `bananadogs`, `krakaziabra mama` to `some name1`, `some description1` ..etc.

File: eventually/tests/unittests/team/testmodel.py (hunk 1 above)
Comment (makarman) [reply eventually/tests/unittests/team/testmodel.py]: done

File: eventually/tests/unittests/team/testmodel.py (hunk 2 above)
Comment (makarman) [reply eventually/tests/unittests/team/testmodel.py]: done

**Review by redmyt — APPROVED** [review]

**Review by skramar — APPROVED** [review]

**Pull request merged**